### PR TITLE
Moved all CI/CD to Travis

### PR DESCRIPTION
**Pull request by rporres**
* Dockerfile was moved to Debian as build is no longer made in Alpine Linux
* helm chart lives now in charts repo: https://github.com/src-d/charts/pull/251

Signed-off-by: Rafael Porres Molina <rafa@sourced.tech>

### CODE REVIEW

**Comment by rporres** [comment]
Deployed in staging correctly https://code-annotation-staging.srcd.run/, although I don't have permissions to further test anything

cc @src-d/applications

**Review by driosalido — COMMENTED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -9,14 +9,49 @@ before_install:
...
+        file_glob: true
+        file: build/*.tar.gz
+        skip_cleanup: true
+        on:
+          tags: true
+          go: 1.9.x
+    - name: 'Deploy to production'
+      stage: release-helm
+      script:
+        - B64_CA_CRT=$PRODUCTION_B64_CA_CRT SERVICE_ACCOUNT_TOKEN=$PRODUCTION_SERVICE_ACCOUNT_TOKEN CLUSTER_ENDPOINT=$PRODUCTION_CLUSTER_ENDPOINT HELM_RELEASE=code-annotatation HELM_CHART=code-annotation K8S_NAMESPACE=default HELM_ARGS="--repo https://src-d.github.io/charts/infra/ --version 0.1.0 --set image.tag=$TRAVIS_TAG -f .helm-production.yml" make deploy
```
Comment (dpordomingo) [inline .travis.yml]:
```suggestion
        - B64_CA_CRT=$PRODUCTION_B64_CA_CRT SERVICE_ACCOUNT_TOKEN=$PRODUCTION_SERVICE_ACCOUNT_TOKEN CLUSTER_ENDPOINT=$PRODUCTION_CLUSTER_ENDPOINT HELM_RELEASE=code-annotation HELM_CHART=code-annotation K8S_NAMESPACE=default HELM_ARGS="--repo https://src-d.github.io/charts/infra/ --version 0.1.0 --set image.tag=$TRAVIS_TAG -f .helm-production.yml" make deploy
```

**Review by dpordomingo — COMMENTED** [review]
question

File: .travis.yml (hunk 1 above)
Comment (rporres) [reply .travis.yml]: good catch!!

**Review by dpordomingo — COMMENTED** [review]
LGTM.
We developed this long time ago, it was a kind of wip and staled more than a year ago, so my level of confidence in this review is 7/10.
Checks I did:
- deploy is done following the same steps in Travis than in former Drone,
- deploy in Staging works, and data is kept between deploys, and the server https://code-annotation-staging.srcd.run is also accessible and running.
- confirmed with Infra that we have snapshots of current production database, and a way to rollback.

I'd recommend:
- merge,
- tag -> deploy
- perform a sanity check to ensure the data is there -> and forget this issue :crossed_fingers:

**Review by dpordomingo — APPROVED** [review]

**Pull request merged**